site/lib/queries.ts

```diff
@@ -20,6 +20,8 @@ export const queries = {
     '*[_type == "post"] | order(publishedAt desc) {summary, "slug": slug.current, title, publishedAt, author->, "imageUrl": mainImage.asset->url}',
   /** fetch all the blog post slugs, for getStaticPaths */
   allPostSlugs: '*[_type == "post"] {"slug": slug.current}',
+  latestPost:
+    '*[_type == "post"] | order(_createdAt desc){"slug": slug.current, title}[0]',
```

Comment:
We should sort by the `publishedAt` field which exists on `Post` documents.
They might not be published in the order that they are created!

Reply:
That's a good point, nice catch!